src/components/accordion/accordion.scss

```diff
@@ -0,0 +1,75 @@
+.accordion-container {
+    height: auto;
+    width: 90%;
+
+    &-heading {
+        font-family: "Poppins", sans-serif;
+        font-size: 24px;
+        line-height: 3rem;
+        text-transform: uppercase;
+        color: #a0a0aa;
+        height: 4.4rem;
+        display: flex;
+        justify-content: space-evenly;
+        align-items: center;
+
+        @media (min-width: 767px) and (max-width: 1023px) {
+            justify-content: center;
+            font-size: 40px;
```

Comment:
This font size needs to be adjusted, definitely for tablet and desktop, potentially for mobile. Currently it is basically the same size as the We're Spacelab title. That title could probably be a tiny bit bigger, but these sub titles definitely need to be smaller.